src/main/java/com/dhivakar/Redirector/Controller/RestService.java

```diff
@@ -10,4 +12,12 @@
     public String  HelloWorld(){
         return "Hello World";
     }
+
+    @RequestMapping("/get/{url}")
+    public RedirectView Redirect(@PathVariable(name="url") String url)
+    {
+        RedirectView redirectView = new RedirectView();
+        redirectView.setUrl("https://www."+url);
```

Comment:
- [ ] Add logic to check given url is a valid url
- [ ] logic to redirect entire url